src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -0,0 +1,5 @@
+databaseChangeLog:
+  - include: 
+      file: db/changelog/changes/01-create-book-table.yaml
+  - include:
+      file: db/changelog/changes/02-add-isDeleted-column-to-book-entity.yaml
```

Comment:
also, I'd rather combine these 2 files in 1 since both of them did not exist before that PR, but that is not that important - so up to you